portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowConstants.java

```diff
@@ -81,6 +81,8 @@
 
 	public static final String LABEL_SCHEDULED = "scheduled";
 
+	public static final String SCHEDULER_INVOCATION = "schedulerInvocation";
+
```

Comment:
This is only used in blogs; can you make this constant private to the BlogsEntryLocalServiceImpl class? Also, it may be a good idea to namespace it, e.g: `_SCHEDULER_INVOCATION = BlogsEntry.class.getName() + "SCHEDULER_INVOCATION"`